doc/pages/regex.asciidoc

```diff
@@ -56,10 +56,11 @@ characters, the `-` character, and the `+` character.
 
 Supported character class escapes are:
 
-* `\d` which matches all digits.
-* `\w` which matches all word characters.
-* `\s` which matches all whitespace characters.
-* `\h` which matches all horizontal whitespace characters.
+* `\d` which matches digits 0-9.
+* `\w` which matches word characters A-Z, a-z, 0-9 and underscore
```

Comment:
It actually matches everything that is accepted by `iswalnum` (which is locale-dependent).
Giving some examples (but not the full list) might still be a good idea:

```
... word characters like `A`, `é`, and `_` (ignoring the `extra_word_chars` option).
```